lib/monitor/run.js

```diff
@@ -16,6 +16,27 @@ var psTree = require('pstree.remy');
 var path = require('path');
 var signals = require('./signals');
 
+function waitForOldKids(oldKids, callback) {
+  if (oldKids) {
```

Comment:
Maybe also check `oldKids.length > 0` ?

Reply:
Ok, changed this line to `if (Array.isArray(oldKids) && oldKids.length > 0) {`